Gameverse/Services/ProductsService.cs

```diff
@@ -139,4 +139,29 @@ public IEnumerable<Review> GetReviewsByProductId(int ProductId)
         return reviews;
     }
 
+    public Product GetRandomGame()
+    {
+        Random random = new Random();
+        var games = _context.Products
+            .Include(p => p.Category)
+            .Where(p => p.Category.CategoryId == 1).ToList();
+        return games.OrderBy(x => Guid.NewGuid()).FirstOrDefault();
+    }
+
+    public Product GetRandomMedia()
+    {
+        var media = _context.Products
+            .Include(p => p.Category)
+            .Where(p => p.Category.CategoryId == 2).ToList();
+        return media.OrderBy(x => Guid.NewGuid()).FirstOrDefault();
+    }
+
+    public Product GetRandomItem()
+    {
+        var items = _context.Products
+            .Include(p => p.Category)
+            .Where(p => (p.Category.CategoryId == 2) || (p.Category.CategoryId == 1)).ToList();
```

Comment:
Daca lasi doar  `_context.Products.ToList()` cred ca ar face acelasi lucru dar o verificare in minus. Il poti lasa si asa ca doar merge